bugbear.py

```diff
@@ -632,16 +634,57 @@ def is_abstract_decorator(expr):
                 isinstance(expr, ast.Attribute) and expr.attr[:8] == "abstract"
             )
 
+        def empty_body(body) -> bool:
+            # Function body consist solely of `pass`, `...`, and/or (doc)string literals
+            return all(
+                isinstance(stmt, ast.Pass)
+                or (
+                    isinstance(stmt, ast.Expr)
+                    and (
+                        # python<3.8
+                        isinstance(stmt.value, (ast.Ellipsis, ast.Str))
+                        or (
+                            # python3.8+
+                            isinstance(stmt.value, ast.Constant)
+                            and (
+                                stmt.value.value is Ellipsis
+                                or isinstance(stmt.value.value, str)
```

Comment:
I'd pull this out as a a new helper, `is_str_or_ellipsis(node)`, and expect the code to get more readable.